### ISSUE TITLE
Do we need to set environment variable `IGNITION_VERSION`?

### ISSUE THREAD

**Issue by wolfv** [conda-forge/libignition-gazebo-feedstock#41]
### Solution to issue cannot be found in the documentation.

- [X] I checked the documentation.

### Issue

In the CMakeLists of "dolly" they check for the env var of `IGNITION_VERSION` to be set to `"fortress"` in order to search for `ignition-gazebo6`.

I was wondering if we should also set that in an activation script? Thoughts?

https://github.com/chapulina/dolly/blob/de9efdf34235d749b9f3b458607d554816304b88/dolly_ignition/CMakeLists.txt#L6-L11

### Installed packages

```shell
-
```

### Environment info

```shell
-
```

**Comment by Tobias-Fischer** [comment]
Yes I think I've seen this elsewhere (the ROS ign packages maybe). Good idea!

**Comment by traversaro** [comment]
In the specific case, `IGNITION_VERSION` is specified by the CI, see https://github.com/chapulina/dolly/blob/6a9b6d698a38cca092901257ca4343d8b2a40176/.github/workflows/ci.yml#L33 . I would be inclined not to define such a variable, for two main reasons:
* Consistency with apt/homebrew packages: I do not think any apt/homebrew package of Modern Gazebo (i.e. what was formerly known as Ignition) defines it
*  As "Modern Gazebo"/Ignition distributions are meant to be side-by-side installable I am not sure setting, I do not think it make sense to have a single env variable. 

However, I would need to check how ros_ign build system works. Perhaps @chapulina herself may have some input on this?

**Comment by wolfv** [comment]
Actually, i saw that in the released version of this package Gazebo5 is hard coded... sadly
